src/main/java/moriyashiine/aylyth/common/registry/AylythBlocks.java

```diff
@@ -0,0 +1,186 @@
+package moriyashiine.aylyth.common.registry;
+
+import com.google.common.base.Suppliers;
+import com.terraformersmc.terraform.sign.block.TerraformHangingSignBlock;
+import com.terraformersmc.terraform.sign.block.TerraformSignBlock;
+import com.terraformersmc.terraform.sign.block.TerraformWallHangingSignBlock;
+import com.terraformersmc.terraform.sign.block.TerraformWallSignBlock;
+import moriyashiine.aylyth.common.block.*;
+import moriyashiine.aylyth.common.data.world.AylythConfiguredFeatures;
+import moriyashiine.aylyth.common.util.AylythUtil;
+import net.fabricmc.fabric.api.object.builder.v1.block.FabricBlockSettings;
+import net.fabricmc.fabric.api.object.builder.v1.block.type.BlockSetTypeBuilder;
+import net.fabricmc.fabric.api.object.builder.v1.block.type.WoodTypeBuilder;
+import net.fabricmc.fabric.api.transfer.v1.item.ItemVariant;
+import net.minecraft.block.*;
+import net.minecraft.block.piston.PistonBehavior;
+import net.minecraft.block.sapling.LargeTreeSaplingGenerator;
+import net.minecraft.block.sapling.SaplingGenerator;
+import net.minecraft.registry.Registries;
+import net.minecraft.registry.Registry;
+import net.minecraft.registry.RegistryKey;
+import net.minecraft.sound.BlockSoundGroup;
+import net.minecraft.util.math.random.Random;
+import net.minecraft.world.gen.feature.ConfiguredFeature;
+import org.jetbrains.annotations.Nullable;
+
+import static net.fabricmc.fabric.api.object.builder.v1.block.FabricBlockSettings.copyOf;
+
+public interface AylythBlocks {
+
+	BlockSetType YMPE_BLOCK_SET_TYPE = new BlockSetTypeBuilder().register(AylythUtil.id("ympe"));
+	BlockSetType POMEGRANATE_BLOCK_SET_TYPE = new BlockSetTypeBuilder().register(AylythUtil.id("pomegranate"));
+	BlockSetType WRITHEWOOD_BLOCK_SET_TYPE = new BlockSetTypeBuilder().register(AylythUtil.id("writhewood"));
+	WoodType YMPE_WOOD_TYPE = new WoodTypeBuilder().register(AylythUtil.id("ympe"), YMPE_BLOCK_SET_TYPE);
+	WoodType POMEGRANATE_WOOD_TYPE = new WoodTypeBuilder().register(AylythUtil.id("pomegranate"), POMEGRANATE_BLOCK_SET_TYPE);
+	WoodType WRITHEWOOD_WOOD_TYPE = new WoodTypeBuilder().register(AylythUtil.id("writhewood"), WRITHEWOOD_BLOCK_SET_TYPE);
+
+	AbstractBlock.Settings STREWN_LEAVES = AbstractBlock.Settings.create().mapColor(MapColor.DARK_GREEN).notSolid().pistonBehavior(PistonBehavior.DESTROY).replaceable();
```

Comment:
This is a mutable field and should be changed to a private method that returns the settings or put inline for each strewn leaves block